signatures/gemss/RedGemss/bench/additional_128_skylake.yaml

```diff
@@ -0,0 +1,6 @@
+platform: Skylake
```

Comment:
Please include all available data (Frequency, compilation options etc; as I did here https://github.com/cryptoeng/pqdb/commit/a5035a1b .) We need this to check out which measurements are comparable. (especially when we implement #69)

(applies to all `bench/` files)